dask/dataframe/tests/test_reshape.py

```diff
@@ -86,6 +86,9 @@ def test_get_dummies_kwargs():
     tm.assert_index_equal(res.columns, pd.Index([1, 2, 3, 5, np.nan]))
 
 
+@pytest.mark.filterwarnings(
+    "ignore::FutureWarning"
```

Comment:
Let's make this ignore a bit more targeted 

```suggestion
    "ignore:In a future version, passing a SparseArray:FutureWarning"
```